src/main/java/com/gmail/inverseconduit/bot/DefaultBot.java

```diff
@@ -34,6 +34,7 @@
     protected final ChatInterface      chatInterface;
 
     protected final Set<CommandHandle> commands = new HashSet<>();
+    protected final Set<CommandHandle> listeners = new HashSet<>();
```

Comment:
Well, this isn't a merge of OakBot into JavaBot, it's a merge of OakBot's "javadoc" command into JavaBot. :P

The reason I added the "listeners" Set is because I need a way to listen for numbers that people type into the chat.